website/static/js/contribAdder.js

```diff
@@ -96,10 +96,17 @@ AddContributorViewModel = oop.extend(Paginator, {
         self.totalPages = ko.observable(0);
         self.childrenToChange = ko.observableArray();
 
+        self.emailSearch = ko.pureComputed(function () {
+            var emailRegex = new RegExp('[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[a-z]+');
```

Comment:
No, wait, I don't think that would work for some cases that QA will almost certainly use. (i.e. nici+testuser1@cos.io) Well maybe.

Still, take a look at http://stackoverflow.com/a/1373724 and maybe do the practical one (include a comment to the link to that answer so people can see where it came from and why you used it)